Use case + user story + UML diagram (#250)
AY2223S1-CS2103T-F12-1/tp: docs/DeveloperGuide.md

|----------|------------------------------------------------|---------------------------------------------------------------------------------------|-------------------------------------------------------------------|
| `* * *`  | property agent                                 | add new buyers                                                                        |                                                                   |
| `* * *`  | property agent                                 | add new properties                                                                    |                                                                   |
| `* *`    | property agent                                 | edit contact and property information offline                                         | keep the information updated                                      |
| `* *`    | property agent                                 | delete existing contact and propety entries                                           | remove redundant information, to keep dataset neat                |

> | `* *`    | property agent with a large number of contacts | sort buyers and properties according to various relevant criteria                     | easily manipulate my list in a way I want to view the information |

Reviewer: ```suggestion
| `* *`    | property agent with a large number of contacts | sort buyers and properties according to various relevant criteria                     | easily find information using their order |
```

2. User finds buyers that are no longer relevant (e.g. already bought a house).
3. User executes delete command on these buyers.

### Use case: Sort properties

**MSS:**

1. User chooses to sort properties by a specified field in a specified order.
2. User executes sort properties command on the currently displayed properties list.

Use case ends.

### Use case: Sort buyers

**MSS:**

1. User chooses to sort buyers by a specified field in a specified order.

> 2. User executes sort buyers command on the currently displayed buyer list.

Reviewer: There should be a step 3 indicating the success outcome

Use case ends.

### Use case: List buyers

**MSS:**

1. User chooses to list buyers

> 2. User executes list buyers command.

Reviewer: 3. Cobb shows a list of all the buyers currently saved?

Author: Oops I accidentally copied this from on top, but i'll help to change that!
